### PR TITLE
Remove1 card

### PR DESCRIPTION
**Pull request by DevSamaa**
Please take a look at the test and method for the CardRemover Class.

### CODE REVIEW

File: BlackJackKata/CardRemover.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+{
+    public class CardRemover
+    {
+        public CardRemover()
+        {
+        }
+
+
+        public List<CardModel> RemoveCard(List<CardModel> incomingDeck)
+        {
```
Comment (sandycurrie) [inline BlackJackKata/CardRemover.cs:14]: Why you return a List? Could it just be a single `CardModel`?

File: BlackJackTests/CardRemoverTests.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                new CardModel("HEART", "Q", 10)
+
+            };
+            yield return new object[] { deckOfCards, deckOfCards.Count };
+
+        }
+
+        [Theory]
+        [MemberData(nameof(DeckOfCards))]
+        public void RemoveCard_GivenADeck_ShouldReturnDeckMinusOne_AndListWithOneCard(List<CardModel> currentDeck, int initialCountOfDeck)
```
Comment (sandycurrie) [inline BlackJackTests/CardRemoverTests.cs]: could try to remove `initialCountOfDeck` and do the count calc inside the test

**Review by sandycurrie — CHANGES_REQUESTED** [review]
I've put a couple suggestions that could be fixed up.

File: BlackJackTests/CardRemoverTests.cs (hunk 2 above)
Comment (DevSamaa) [reply BlackJackTests/CardRemoverTests.cs]: I fixed this up.

File: BlackJackKata/CardRemover.cs (hunk 1 above)
Comment (DevSamaa) [reply BlackJackKata/CardRemover.cs:14]: I'm thinking about changing this whole bit later on.

**Review by sandycurrie — APPROVED** [review]
Looks great!

**Pull request merged**